src/W/Message.elm

```diff
@@ -160,6 +165,20 @@ danger =
             { attrs | color = Theme.dangerForeground }
 
 
+borderWidth : Int -> Attribute msg
+borderWidth v =
+    Attribute <|
+        \attrs ->
+            { attrs | borderWidth = v }
+
+
+noBorder : Attribute msg
```

Comment:
Public functions need doc comments (even if they're empty). So you should add `{-| -}`   in the line above the function name.